salt/grains/nxos.py

```diff
@@ -22,18 +22,17 @@
 
 
 def __virtual__():
-    try:
-        if salt.utils.platform.is_proxy() and __opts__['proxy']['proxytype'] == 'nxos':
-            return __virtualname__
-    except KeyError:
-        pass
-
-    return False
+    return __virtualname__
```

Comment:
Is this for the native support?

Reply:
That's correct.  We need to support grains for proxy and native minions.